larq_compute_engine/core/bgemm_trmul_params.h

```diff
@@ -35,21 +35,20 @@ void PopulateBinaryTrMulParams(ruy::TrMulParams* params) {
     return;
   }
 
-  using PackedTBitpacked = ruy::PackedType<ThePath, TBitpacked>;
   using Kernel = BgemmKernel<ThePath, DstScalar, MulParamsType>;
   using LhsKernelLayout = typename Kernel::LhsLayout;
   using RhsKernelLayout = typename Kernel::RhsLayout;
 
   params->path = ThePath;
 
-  CreatePackedMatrix<TBitpacked, PackedTBitpacked>(
+  CreatePackedMatrix<TBitpacked, TBitpacked>(
       Side::kLhs, ToKernelLayout<LhsKernelLayout>(), params);
-  CreatePackedMatrix<TBitpacked, PackedTBitpacked>(
+  CreatePackedMatrix<TBitpacked, TBitpacked>(
       Side::kRhs, ToKernelLayout<RhsKernelLayout>(), params);
```

Comment:
What does this change do? What is normally the 'packed' type of `float` or `int8`, and why is it `TBitpacked` in our case?

Reply:
It seems that in all cases, the 'packed type' is always just equal to the original type: https://github.com/google/ruy/blob/f876353d64a03e0e1b5d2262c0accbc5011bda3b/ruy/pack_common.h#L47-L53

I *think* that the distinction between a packed type and a normal type exists so that it's possible to take uint8 data and convert it to int8 as part of the packing process, for example. It's weird though because as far as I can tell that functionality doesn't actually get used anywhere in Ruy.

In any case, we definitely want to keep the `TBitpacked` type when we perform the packing.

Reply:
Thanks :+1: